simapp/v2/simdv2/cmd/commands.go

```diff
@@ -38,6 +38,8 @@ import (
 	genutilcli "github.com/cosmos/cosmos-sdk/x/genutil/client/cli"
 	genutiltypes "github.com/cosmos/cosmos-sdk/x/genutil/types"
 	v2 "github.com/cosmos/cosmos-sdk/x/genutil/v2/cli"
+	"cosmossdk.io/client/docs"
```

Comment:
This import doesn't exist and the ordering is wrong (Make lint-fix should have fixed that.)